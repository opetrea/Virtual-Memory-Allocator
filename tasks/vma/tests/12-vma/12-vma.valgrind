==91869== Memcheck, a memory error detector
==91869== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91869== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91869== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91869== Parent PID: 91868
==91869== 
==91869== 
==91869== HEAP SUMMARY:
==91869==     in use at exit: 0 bytes in 0 blocks
==91869==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==91869== 
==91869== All heap blocks were freed -- no leaks are possible
==91869== 
==91869== For lists of detected and suppressed errors, rerun with: -s
==91869== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
