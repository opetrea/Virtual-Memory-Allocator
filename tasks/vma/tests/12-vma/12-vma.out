This zone was already allocated.
Total memory: 0x14 bytes
Free memory: 0xA bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x5 - 0xF
Miniblock 1:		0x5		-		0xF		| RW-
Block 1 end
